internal/internal_workflow_client.go

```diff
@@ -1048,6 +1048,14 @@ func (wc *workflowClient) getWorkflowStartRequest(
 		return nil, errors.New("Invalid JitterStart option")
 	}
 
+	firstRunAtTimestamp := options.FirstRunAt.UnixNano()
+	if options.FirstRunAt.IsZero() {
+		firstRunAtTimestamp = 0
+	}
+	if firstRunAtTimestamp < 0 {
+		return nil, errors.New("Invalid FirstRunAt option")
```

Comment:
please cover these negative cases in unit tests

Reply:
Done